tools/tests/test_base.py

```diff
@@ -288,21 +332,74 @@ def isDead(self, pid, timeout=5):
         return False
 
 
+'''Determine the last build osqueryi.exe'''
```

Comment:
nit, this is above the daemon check and uses a different comment style than the one below